addons/mail/tests/test_mail_render.py

```diff
@@ -162,26 +158,26 @@ def test_evaluation_context(self):
             'custom_value': 'Custom Render Value'
         }
         srces = [
-            '<b>I am ${user.name}</b>',
-            '<span>Datetime is ${format_datetime(datetime.datetime(2021, 6, 1), dt_format="MM - d - YYY")}</span>',
-            '<span>Context ${ctx.get("custom_ctx")}, value ${custom_value}</span>',
+            '<b>I am {{ user.name }}</b>',
+            '<span>Datetime is {{ format_datetime(datetime.datetime(2021, 6, 1), dt_format="MM - d - YYY") }}</span>',
+            '<span>Context {{ ctx.get("custom_ctx") }}, value {{ custom_value }}</span>',
         ]
         results = [
             '<b>I am %s</b>' % self.env.user.name,
             '<span>Datetime is 06 - 1 - 2021</span>',
             '<span>Context Custom Context Value, value Custom Render Value</span>'
         ]
         for src, expected in zip(srces, results):
-            for engine in ['jinja']:
+            for engine in ['inline_template']:
                 result = MailRenderMixin.with_context(**custom_ctx)._render_template(
                     src, partner._name, partner.ids,
                     engine=engine, add_context=add_context
                 )[partner.id]
                 self.assertEqual(expected, result)
 
     @users('employee')
-    def test_render_mail_template_jinja(self):
-        template = self.env['mail.template'].browse(self.test_template_jinja.ids)
+    def test_render_field(self):
```

Comment:
``test_render_mail_template_fields`` (mail template is not the sole model having fields + rendering :) )